pkg/config/config.go

```diff
@@ -30,6 +30,13 @@ const (
 	WebhookCertDir        = "/tmp/k8s-webhook-server/serving-certs"
 )
 
+const (
+	LB_SERVICE_SIZE_SMALL  = "SMALL"
```

Comment:
Please follow Golang convention:
LB_SERVICE_SIZE_SMALL -> LBSizeSmall

And where the values are used? I just wonder if we should use "SMALL" or "Small" or "small".

Reply:
If the values come from YAML/CRD, we should use "Small", "Medium", ...

Reply:
Removed this section since it is already set in nsxt sdk. Added `GetServiceSize` func in config.go